### PR TITLE
Version with class

### PR DESCRIPTION
**Pull request by jekyllbot**
This PR intends to normalize the way we document jekyll plugins release process:

https://jekyllrb.com/docs/maintaining/releasing-a-new-version/#for-non-core-gems

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ inherit_gem:
   jekyll: .rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.4
```
Comment (DirtyF) [inline .rubocop.yml]: As they were no differences for this plugin, I enforced 2.4.

Comment (pathawks) [reply .rubocop.yml]: But won't this allow Rubocop to make suggestions that would be incompatible with Ruby 2.3? I don't think we want this.

Comment (DirtyF) [reply .rubocop.yml]: Fair. Reverted.

**Comment by DirtyF** [comment]
@jekyllbot: :shipit: +dev

**Pull request merged**